core/domain/exp_domain.py

```diff
@@ -468,8 +481,8 @@ def create_default_exploration(
         taken from feconf; 'tags' and 'param_changes_list' are initialized to
         empty list; 'states_schema_version' is taken from feconf; 'states_dict'
         is derived from feconf; 'param_specs_dict' is an empty dict; 'blurb' and
-        'author_notes' are initialized to empty string; 'version' is
-        initializated to 0.
+        'author_notes' are initialized to empty string; 'version' and
+        image_counter is initializated to 0.
```

Comment:
Make it 'image_counter'. (use '')

Reply:
done.

Reply:
I don't see any changes here, let me know if my comment is not clear.